packages/opentelemetry-web/package.json

```diff
@@ -6,13 +6,13 @@
   "types": "build/src/index.d.ts",
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
-    "check": "gts check",
+    "lint": "gts check",
```

Comment:
Minor: Could you please keep `lint` and `lint:fix` scripts together (in alphabetical order).

Reply:
done